.github/labeler.yml

```diff
@@ -0,0 +1,2 @@
+More Information Required:
+    - '^((?!Steps To Reproduce)[\s\S])*$|^((?!Repository to reproduce)[\s\S])*$|^((?!Expected result)[\s\S])*$|^((?!Actual result)[\s\S])*$|^((?!CLI Version)[\s\S])*$|^((?!plugin Version)[\s\S])*$|^((?!OS)[\s\S])*$'
```

Comment:
That is a lot of negative lookups! How did you test this?

Doing something like the following works when I remove some of the pieces of the string:
```javascript
'Steps To Reproduce\nRepository to reproduce\nExpected result\nActual result\nCLI Version\nplugin Version\nOS'.match(/^((?!Steps To Reproduce)[\s\S])*$|^((?!Repository to reproduce)[\s\S])*$|^((?!Expected result)[\s\S])*$|^((?!Actual result)[\s\S])*$|^((?!CLI Version)[\s\S])*$|^((?!plugin Version)[\s\S])*$|^((?!OS)[\s\S])*$/);
```

But something like the following will not work when I remove some of the peices:
```javascript
'Steps To Reproduce\nRepository to reproduce\nExpected result\nActual result\nCLI Version\nplugin Version\nOS'.match('^((?!Steps To Reproduce)[\s\S])*$|^((?!Repository to reproduce)[\s\S])*$|^((?!Expected result)[\s\S])*$|^((?!Actual result)[\s\S])*$|^((?!CLI Version)[\s\S])*$|^((?!plugin Version)[\s\S])*$|^((?!OS)[\s\S])*$');
```

According to [this line](https://github.com/github/issue-labeler/blob/master/src/main.ts#L170), I don't think it will mark yours as a regexp. Even if you add `/` here, I still get mixed results. For example, the following does not work which is what the code looks like it is doing.

```javascript
const match = '/^((?!Steps To Reproduce)[\s\S])*$|^((?!Repository to reproduce)[\s\S])*$|^((?!Expected result)[\s\S])*$|^((?!Actual result)[\s\S])*$|^((?!CLI Version)[\s\S])*$|^((?!plugin Version)[\s\S])*$|^((?!OS)[\s\S])*$/'.match(/^\/(.+)\/(.*)$/);
const regexp = new RegExp(match[1], match[2]);
// Will not work when I start removing peices
'Steps To Reproduce\nRepository to reproduce\nExpected result\nActual result\nCLI Version\nplugin Version\nOS'.match(regexp);
```

Reply:
Im gonna try to improve this expression...